website/astro.config.ts

```diff
@@ -397,11 +362,10 @@ export default defineConfig({
 
 	markdown: {
 		syntaxHighlight: "prism",
-		remarkPlugins: [remarkToc],
-		rehypePlugins: [rehypeSlug, [rehypeAutolinkHeadings, autolinkConfig]],
+		rehypePlugins: [rehypeSlug, ...rehypeAutolink()],
 	},
 
-	adapter: netlifyStatic(),
+	adapter: netlify(),
 
 	vite: {
```

Comment:
For some reason Vite is complaining about some imported code by Astro's DevToolbar. Supposedly `vite.server.fs.allow` the path used should fix it, but didn't work for me.

Reply:
We have to disable their remote image service, because they don't support multiple remote image domains from the config file. Their alternative is to use another configuration file

Reply:
There we go, this the `vite.server.fs.allow` should be fixed, Vite now has a utility to allow workspace packages to be served, which fixes the issue with serving files from dependencies.